Diff/diff.py

```diff
@@ -1,14 +1,19 @@
-import codecs
+from __future__ import annotations
+
 import difflib
-import os.path
+import os
 import time
 
 import sublime
 import sublime_plugin
 
+import typing
```

Comment:
Are you following any logic in regards to grouping imports? Feels like `typing` should be grouped together with other standard libraries.